process.sh

```diff
@@ -38,9 +38,9 @@ main() {
         # Exclude the *HUGE* PNGs in the base directory:
         # The tiles for S3 are in subdirectories;
         # We keep the PNGs around because it takes a long time to generate them.
-        TILES_BASE='vanderbilt.images'
+        TILES_BASE='vanderbilt/vanderbilt.images'
         aws s3 cp --exclude "$OUTPUT/*.ome.tif*" --recursive "$OUTPUT" s3://"$S3_TARGET"
-        gsutil cp "$OUTPUT/vanderbilt/$TILES_BASE/*.ome.tif*" gs://vitessce-data/$TILES_BASE
+        gsutil -m cp -r "$OUTPUT/$TILES_BASE/" gs://"$S3_TARGET/$TILES_BASE/"
```

Comment:
I'd rather keep all the explicit gsutil commands to one line. 

`gsutil -m cp -r "$OUTPUT/vanderbilt/vanderbilt.images" gs://"$S3_TARGET/vanderbilt/vanderbilt.images"`